build.gradle

```diff
@@ -49,7 +49,12 @@ subprojects { subProject ->
   spotless {
     kotlin {
       target "**/*.kt"
-      ktlint(dep.ktlintVersion).userData(['indent_size': '2', 'continuation_indent_size': '4'])
+      ktlint(dep.ktlintVersion).userData(
+              [
+                      'indent_size': '2',
+                      'continuation_indent_size': '4',
+                      'disabled_rules': 'import-ordering'
```

Comment:
sigh, I had to disable this because they now do the "default intellij" style of putting java/javax/kotlin after other imports, but that's not our style

Reply:
If you want to change it. I'm game...